evidence/kianenigma/0002-2024-12-retention-rank-4.md

```diff
@@ -0,0 +1,53 @@
+| **Report Date** | Date of submission (2024/12/01)                                                             |
+| **Submitted by**| `kianenigma`
+
+## Member details
+
+- Matrix username: @kianenigma
+- Polkadot address: `1eTPAR2TuqLyidmPT9rMmuycHVm9s9czu78sePqg2KHMDrE`
+- Current rank: 4
+- Date of initial induction: seeding
+- Date of last report: [22/05/2024](https://github.com/polkadot-fellows/Evaluations/pull/28)
+- Area(s) of Expertise/Interest: 
+    - FRAME
+    - Staking / Inflation
+    - JAM
+- Status: **Inactive / Part-time** (salary divided by half)
+
+## Reporting period
+
+- Start date: 2024/06
+- End date: 2024/11
+
+
+## Evidence
+
+* Maintenance of FRAME ([eg](https://github.com/paritytech/polkadot-sdk/pull/5995)), Polkadot-SDK templates ([eg](https://github.com/paritytech/polkadot-sdk/issues/5242)), and new components such as the [`OmniNode`](https://github.com/orgs/paritytech/projects/157/views/1) and its [api docs](https://github.com/paritytech/polkadot-sdk/pull/6094).
+* Abundant mentorship either through position at Parity, and or participating (free of charge) in PBA(X) program.
+* Ample reviews on the Staking system ([e.g.](https://github.com/paritytech/polkadot-sdk/pull/6034)) + [Inflation](https://github.com/polkadot-fellows/runtimes/pull/443) system.
+* While bandwidth is limited, I am attempting to guide developers into [materializing OPF](https://github.com/polkadot-fellows/runtimes/issues/375), which has passed WFC track.
+* Retention and dissemination of core Polkadot knowledge, including JAM:
+    * Demystifying JAM - [Blog](https://blog.kianenigma.com/posts/tech/demystifying-jam/) / [Presentation](https://blog.kianenigma.com/posts/tech/demystifying-jam/presentation/), which has been featured in many places, including [`@govofyork`'s X account](https://x.com/gavofyork/status/1860633087136178639), [Parity's blog](https://www.parity.io/blog/JAM-demystified-explainer), and [Polkadot X account](https://x.com/Polkadot/status/1861008901606391826), and 
+    * [Blockchain Scalability](https://blog.kianenigma.com/blockchain-reimagined/presentation-tum/) - TUM Munich Blockchain Conference
+    * [Polkadot-SDK Features Overview in 2024](https://blog.kianenigma.com/talks/frame-2024/)
+* Exploring new FRAME ideas such as [FLITE](https://github.com/kianenigma/flite), which have been a [source of inspiration](https://github.com/OpenZeppelin/polkadot-runtime-templates/issues/257#issuecomment-2278749395) for [OpenZeppelin templates](https://github.com/OpenZeppelin/polkadot-runtime-templates/pull/300). 
+  
+
+## Voting record
+*Provide your voting record in relation to required thresholds for your rank.* 
+
+|  Ranks | Activity thresholds | Agreement thresholds | Member's voting activities | Comments |
+|---|---|---|---|---|
+|IV|48%   |100%  | 51 votes out of 107, leading to a 48% rate, starting from ref 148 onwards, which was my retention in June  | |
```

Comment:
|IV| 60% | 90% | 51 votes out of 107, leading to a 48% rate, starting from ref 148 onwards, which was my retention in June |  |

Reply:
Please do not edit the thresholds. Only fill in the 'Member's voting activities" and "Comment" columns with your data.